src/materialized/ci/README.md

```diff
@@ -6,21 +6,21 @@ The Materialize Emulator is an all-in-one Docker image available on Docker Hub,
 | **Production deployments** | ❌ Not suitable due to performance and license limitations.                 | ✔️
 | **Performance**            | ❌ Limited. Services are bundled in a single container.                     | ✔️ High. Services are scaled across many machines.                           |
 | **Dedicated Support**      | ❌                                                                          | ✔️                                                                           |
-| **Sample data**            | ✔️ Quickstart data source                                                   | ✔️ Quickstart data source                                                    |
-| **Data sources**           | ✔️ Connect using SQL configuration                                          | ✔️ Connect using a streamlined GUI                                           |
-| **Version upgrades**       | ✔️ Manual, with no data persistence                                         | ✔️ Automated, with data persistence                                          |
+| **Sample data**            | ✔️ Quickstart data source                                                    | ✔️ Quickstart data source                                                    |
+| **Data sources**           | ✔️ Connect using a streamlined GUI                                           | ✔️ Connect using a streamlined GUI                                           |
+| **Version upgrades**       | ✔️ Manual, with no data persistence                                          | ✔️ Automated, with data persistence                                          |
 | **Use case isolation**     | ❌                                                                          | ✔️                                                                           |
 | **Fault tolerance**        | ❌                                                                          | ✔️                                                                           |
 | **Horizontal scalability** | ❌                                                                          | ✔️                                                                           |
-| **GUI**                    | ❌                                                                          | ✔️ Materialize Console                                                       |
+| **GUI**                    | ✔️ Materialize Console (http://localhost:6874)                               | ✔️ Materialize Console                                                       |
 
 ## Usage
 
 To launch the Docker container:
 
 ```
 docker pull materialize/materialized:latest
-docker run -d -p 127.0.0.1:6875:6875 -p 127.0.0.1:6876:6876 materialize/materialized:latest
+docker run -p 127.0.0.1:6874:6874 -p 127.0.0.1:6875:6875 -p 127.0.0.1:6876:6876 materialize/materialized:latest
```

Comment:
Why remove `-d`

Reply:
Put it back, I personally find it easier to run stuff in the foreground.